pkg/config/wikidata.go

```diff
@@ -67,9 +71,10 @@ var wikidata = struct {
 	gzip:         "Q27824060",
 	tar:          "Q283579",
 	warc:         "Q10287816",
-	definitions:  "wikidata-definitions-1.0.0",
+	definitions:  "wikidata-definitions-2.x.x",
```

Comment:
This will be `2.0.0` through  the IRI work and this should be in the same release and so 2.0.0 should work well for this.